CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-alpha.1
+
+<!-- generated comparing v6.0.0-alpha.0..next -->
+
+_Apr 3, 2024_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔥 Converted 3 Material UI components to support static CSS extraction. The progress is at 36%!
+- 🚀 Added a spacing CSS variable to Material UI and Joy UI theme.
```

Comment:
```suggestion
- 🚀 Added a spacing CSS variable to Material UI and Joy UI theme.
```

Reply:
```suggestion
- 🚀 Added spacing CSS variables to the Material UI and Joy UI themes.
```

Just building off of Marija's suggestion